modules/images-configuration-registry-mirror.adoc

```diff
@@ -0,0 +1,182 @@
+requested from the source repository.
+
+The mirror information you enter is added to the `/etc/containers/registries.conf`
+file on every node in the {product-title} cluster.
+
+When a node makes a request for an image from the source repository, it
+tries each mirrored repository in turn until it finds the requested on content.
+If they all fail, the cluster tries the source repository. Upon success, the
+image is pulled to the node.
+
+.Prerequisites
+* Have administrative privilege to a working {product-title} cluster.
+
+.Procedure
+
+. Configure mirrored repositories. To do that, you can either:
++
+* Set up a mirrored repository with Red Hat Quay, as
+described in link:https://access.redhat.com/documentation/en-us/red_hat_quay/3/html/manage_red_hat_quay/repo-mirroring-in-red-hat-quay[Red Hat Quay Repository Mirroring].
+Using Red Hat Quay allows you to copy images from one repository to another,
+but also automatically sync those repositories repeatedly over time.
+* Use a tool such as `skopeo` to copy images manually
+from the source directory to the mirrored repository.
++
+For example, after installing the skopeo RPM package
+on a RHEL 7 or RHEL 8 system, use the `skopeo` command as shown in this example:
++
+----
+$ skopeo copy \
+   docker://registry.access.redhat.com/ubi8/ubi-minimal@sha256:c505667389712dc337986e29ffcb65116879ef27629dc3ce6e1b17727c06e78f \
+   docker://example.io/ubi8/ubi-minimal
+----
++
+This example assumes you have a container image registry named
+`example.io` with an image repository named `example` to which
+you want to copy the `ubi8/ubi-minimal` image from
+`registry.access.redhat.com`.
+This will allow you to tell your {product-title}
+cluster to redirect requests made of the source repository to the
+mirrored repository.
+
+. Log into your {product-title} cluster.
+
+. Create an ImageContentSourcePolicy file (for example,
+`registryrepomirror.yaml`), replacing the source and
+mirrors with those of your own registry/repository pairs and images
+
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: ubi8repo
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - example.io/example/ubi-minimal<1>
+    source: registry.access.redhat.com/ubi8/ubi-minimal<2>
+  - mirrors:
+    - example.com/example/ubi-minimal
+    source: registry.access.redhat.com/ubi8/ubi-minimal
+~                                                 
+----
+<1> Indicates the name of the image registry/repository
+<2> Indicates the registry/repository containing the content being mirrored
+
+. Create the new ImageContentSourcePolicy:
++
+----
+$ oc create -f registryrepomirror.yaml
+----
++
+Once the ImageContentSourcePolicy has been created,
+the new settings will be deployed to each node
+and shortly start using the mirrored repository
+for requests to the source repository.
+
+. To check that the mirrored configuration worked,
```

Comment:
I'd make these next three steps a single step and substeps.

Reply:
Done.